vowpalwabbit/parse_example_json.h

```diff
@@ -640,16 +751,40 @@ class DefaultState : public BaseState<audit>
       if (ctx.key_length == 6 && !strcmp(str, "_multi"))
         return &ctx.multi_state;
 
+      if (ctx.key_length == 3 && !strcmp(str, "_df"))
+        return &ctx.df_state;
+
       if (ctx.key_length == 4 && !_stricmp(ctx.key, "_tag"))
         return &ctx.tag_state;
 
+      if (ctx.key_length == 4 && !_stricmp(ctx.key, "_inc"))
```

Comment:
is there any specific reason for using ctx.key in some places and str in others?

Reply:
I don't believe there is a reason, especially since it is set at the top of the function